frontend/site-registry/src/app/features/details/SaveSiteDetailsSlice.ts

```diff
@@ -27,8 +27,9 @@ const initialState: SaveSiteDetails = {
 
 export const saveSiteDetails = createAsyncThunk(
   'saveSiteDetails',
-  async (siteDetailsDTO: any, { getState }) => {
+  async (siteId: string, { getState }) => {
     const saveDTO = getSiteDetailsToBeSaved(getState());
+    saveDTO.siteId = siteId;
```

Comment:
We already have site we dont't need to pass siteId to it. Please update the code to have no parameters being passed to this function.